src/include/catalog/postgres/pg_defs.h

```diff
@@ -0,0 +1,6 @@
+#pragma once
+
+namespace terrier::catalog::postgres {
+
+const uint32_t MAX_NAME_LENGTH = 63;
```

Comment:
Move to [postgres_defs.h](https://github.com/cmu-db/terrier/blob/master/src/include/network/postgres/postgres_defs.h)